### PR TITLE
Add progressbar

### PR DESCRIPTION
**Pull request by mfrachet**
Add a Progress bar indicator

Example on: https://design-system-git-add-progressbar-strapijs.vercel.app/?path=/story/progressbar--m

## API

```jsx
<ProgressBar value={100} size="S">
   100/200 plugins loaded
</ProgressBar>
```

## Voiceover

Screen reader usage of the progress bar on Safari

### CODE REVIEW

File: packages/strapi-design-system/src/ProgressBar/ProgressBar.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  value: 0,
+  size: 'M',
+};
+
+ProgressBar.propTypes = {
+  min: PropTypes.number,
+  max: PropTypes.number,
+  value: PropTypes.number,
+  children: PropTypes.string.isRequired,
+  size: PropTypes.oneOf(['S', 'M']),
```
Comment (HichamELBSI) [inline packages/strapi-design-system/src/ProgressBar/ProgressBar.js:56]: @maevalienard, for components sizes, sometimes we have `S, L, XL` and sometimes `S, M`. WDYT about using one way to make it more consistent. 😃

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

File: packages/strapi-design-system/src/ProgressBar/ProgressBar.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+ProgressBar.defaultProps = {
+  min: 0,
+  max: 100,
+  value: 0,
+  size: 'M',
+};
+
+ProgressBar.propTypes = {
+  min: PropTypes.number,
```
Comment (soupette) [inline packages/strapi-design-system/src/ProgressBar/ProgressBar.js:52]: Do you mind ordering the prop-types ? I find it easier to read... Can you also add the following [rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-prop-types.md) to the linter,

Comment (mfrachet) [reply packages/strapi-design-system/src/ProgressBar/ProgressBar.js]: Done, thanks 😊

File: packages/strapi-design-system/.eslintrc.js [hunk 3]
```diff
@@ -15,4 +15,7 @@ module.exports = {
     'plugin:react/recommended', // Uses the recommended rules from @eslint-plugin-react
     'plugin:prettier/recommended', // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
   ],
+  rules: {
```
Comment (soupette) [inline packages/strapi-design-system/.eslintrc.js:18]: 👍

**Review by maevalienard — APPROVED** [review]
LGTM thank you! :)

File: packages/strapi-design-system/src/ProgressBar/ProgressBar.js (hunk 1 above)
Comment (maevalienard) [reply packages/strapi-design-system/src/ProgressBar/ProgressBar.js:55]: Oh wow 😂  So many weird things even when you try to make things clean. This is perfect, thanks!

Comment (HichamELBSI) [reply packages/strapi-design-system/src/ProgressBar/ProgressBar.js:55]: No worries !! Thanks @maevalienard

Comment (mfrachet) [reply packages/strapi-design-system/src/ProgressBar/ProgressBar.js:55]: What's the prefered way?

**Review by soupette — APPROVED** [review]
LGTM

**Review by HichamELBSI — APPROVED** [review]
LGTM

**Pull request merged**